spec/services/sla_breach_service_spec.rb

```diff
@@ -107,23 +109,109 @@
         Timecop.return
       end
 
-      context "on Monday @10:05am UTC"
-      it 'returns a hash of total SLA breaches of attention_needed_breach_since by vita_partner_id' do
-        expect(subject.breach_threshold).to eq(Time.utc(2021, 2, 3, 10, 5)) # Wednesday 2/3/21 @ 10:05am UTC
-        expect(subject.attention_needed_breach).to eq(
-          {
+      context "on Monday @10:05am UTC" do
+        it 'returns a hash of total SLA breaches of attention_needed_breach_since by vita_partner_id' do
+          expect(subject.breach_threshold).to eq(Time.utc(2021, 2, 3, 10, 5)) # Wednesday 2/3/21 @ 10:05am UTC
+          expect(subject.attention_needed_breaches).to eq(
+            {
               vita_partner_1.id => 1,
               vita_partner_2.id => 2
+            }
+          )
+        end
+      end
+
+      context "on Monday @ 11:25am UTC" do
+        it "trips the 10:55am client into SLA breach" do
+          t = Time.utc(2021, 2, 6, 0, 0, 0) # 2/6/21
+          Timecop.freeze(t.next_occurring(:monday) + 11.hours + 25.minutes) do
```

Comment:
Just learned that we've had some issues with Timecop in the past - might be worth a discussion